### PR TITLE
Fix for DF-1567

### PR DESCRIPTION
**Pull request by sebworks**
Add rule and adjust spacing on mobile for 'Doing Business with Us' pages.

## Changes

- Updates 'doing-business-with-us/index.html' file to add rule and adjust spacing on mobile.
- Updates 'static/css/main.less' file to include new less file.
- Adds 'static/css/business.less' file to add rule and adjust spacing on mobile. 

## Review

- @jimmynotjim 
- @anselmbradford 

et al

## Preview
screen shot 2015-03-13 at 12 52 05 pm

### CODE REVIEW

File: doing-business-with-us/index.html [hunk 1]
```diff
@@ -23,12 +23,14 @@
         <a href="mailto:david.gragan@cfpb.gov">Contact our head of procurement.</a>
     </p>
 
-    <section class="block block__bg block__border-top block__border-bottom block__flush-sides">
+    <section class="block block__bg block__border-top block__border-bottom
```
Comment (jimmynotjim) [inline doing-business-with-us/index.html]: These could be stacked for easier skimming.

**Comment by jimmynotjim** [comment]
Side note, those step icons look like they aren't true circles and the digit isn't centered. Might be worth adding as a new task though because it's out of scope of this one.

**Comment by sebworks** [comment]
@jimmynotjim, I created https://github.com/cfpb/cfgov-refresh/issues/363.

**Comment by jimmynotjim** [comment]
LTGM

**Pull request merged**